sdk/ml/azure-ai-ml/azure/ai/ml/entities/__init__.py

```diff
@@ -40,7 +40,8 @@
 from ._deployment.scale_settings import DefaultScaleSettings, TargetUtilizationScaleSettings
 from ._endpoint.batch_endpoint import BatchEndpoint
 from ._endpoint.endpoint import Endpoint
-from ._endpoint.online_endpoint import KubernetesOnlineEndpoint, ManagedOnlineEndpoint, OnlineEndpoint
+from ._endpoint.online_endpoint import KubernetesOnlineEndpoint, ManagedOnlineEndpoint, OnlineEndpoint,\
```

Comment:
Is the "\" expected?

Reply:
Yes it is. I had to shorten the line otherwise pylint errors show up.